providers/shared/components/directory/id.ftl

```diff
@@ -0,0 +1,112 @@
+[#ftl]
+
+[@addComponentDeployment
+    type=DIRECTORY_COMPONENT_TYPE
+    defaultGroup="solution"
+/]
+
+[@addComponent
+    type=DIRECTORY_COMPONENT_TYPE
+    properties=
+        [
+            {
+                "Type"  : "Description",
+                "Value" : "A managed directory services instance"
+            }
+        ]
+    attributes=
+        [
+            {
+                "Names" : "Engine",
+                "Types" : STRING_TYPE,
+                "Values" : [ "simple" ],
+                "Mandatory" : true
+            },
+            {
+                "Names" : "EnableSSO",
+                "Types" : BOOLEAN_TYPE,
+                "Default" : false
+            },
```

Comment:
This might be better as an AWS specific parameter? This is really for SSO into the console right?

Reply:
I read this as using the AD as an SSO for other functions within the provider. I don't believe Azure allows you to set this as false, but AWS and other providers probably would so an "anything but Azure" specific parameter, which is why I went global. Thoughts?

Reply:
I think this is an AWS only thing. The SSO integration on their directory means that users created with the directory can log into the AWS Console 

With Azure AD DS  ( Their directory servcies implementation ) it works the other way around. Azure AD users can login to computers that on the Azure AD DS Domain. 

So this SSO is just for AWS